yarn-project/sequencer-client/src/sequencer/abstract_phase_manager.ts

```diff
@@ -479,9 +477,8 @@ function patchPublicStorageActionOrdering(
   execResult: PublicExecutionResult,
   phase: PublicKernelPhase,
 ) {
-  const { publicDataReads, publicDataUpdateRequests } = PhaseIsRevertible[phase]
-    ? publicInputs.end
-    : publicInputs.endNonRevertibleData;
+  const { publicDataUpdateRequests } = PhaseIsRevertible[phase] ? publicInputs.end : publicInputs.endNonRevertibleData;
```

Comment:
Not worth in this PR I think, but patching the reads is no longer dependent on the phase, so might be better to extract to its own function.

Reply:
That's a good point! This will be deleted and done in the circuit eventually, so I'll just leave it as is 😛